packages/clean-tag/package.json

```diff
@@ -12,7 +12,9 @@
   "author": "Brent Jackson",
   "license": "MIT",
   "dependencies": {
-    "html-tags": "^2.0.0",
+    "html-tags": "^2.0.0"
+  },
+  "peerDependencies": {
     "react": "^16.8.5",
```

Comment:
Thanks for this! I think the versions in peer dependencies should probably be as loose as possible – IIRC versions 1-3 of styled-system and `>=16.3` for react